existing-communities/README.md

```diff
@@ -1 +1,47 @@
+
+**Communal Communities**  
+Communal communities center their ideals around the contact between communities members{{"meijering2007intentional" | cite}}. This means that the platform (chat-groups, website, etc.) or place (offices, pub, etc.) plays a central key in this type of community. The place or subject of the platform focuses on the particular characteristic that the members have in common, like a cooking-forum for people who like cooking or a chat-group for parents whos kids are in the same class.
+
+**Religious Communities**  
+Religious communities have their ideals cntered around a belief or religion. If you look at these communities based on the religions, then there is no boundry to the location of the community. However, there are religious communities that also behave like a communal community, like local church-communities. The ideals of the religion are the ideals of the community.
+
+**Practical Communities**  
+Practical communities are special because they don't center around ideals at all. Practical communities arise out of pure convenience, when people are better of sharing information or resources (on a regulare basis). This can be a community of farmers sharing farming equipment, or students sharing a house.
+
+### MMO-RPG Communities
+Massive Multiplayer Online Role-Playing Games are, like the name describes, online games where the player explores a virtual world as one of many type of characters and can interact with an enormous amount of other players. Popular MMO-RPG's often have two types of communites: communities outside of the game (fans and forums) and communities inside of the game (parties and guilds). For this research, we will be looking at the communities inside the game, because the dynamics of these communities often require active participation.  
+Most MMO-RPG's have two in-game systems that can be seen as communities: parties and guilds. A party is a small group of players that train or do quests (in-game missions) together for a short period of time (possibly one game session). The benefit of forming a party is experience-sharing (to some degree) and players can kill high-level in-game creatures they could kill on their own. If a party is formed for the purpose of doing a quest, the composition of the party (the distribution of roles or functions) may be addapted towards the goal of the quest to ensure optimal performance{{"chen2009communication" | cite}}.  
+A guild is a group of players forming a community, for creating a social connection or to make it easier to form a trust-worthy or well-composed party over a longer period of time (longer than one game session){{"rossi2008mmorpg" | cite}}{{"kelly2004play" | cite}}{{"verhagen2009demystifying" | cite}}. Within a guild, members can be classified based on connectivity, type of player, role within the guild and involvement{{"ang2010social" | cite}}.
```

Comment:
`community, for creating`

This isn't grammatically correct I think. It should be something like `community, with the purpose of` or the comma should be removed. Also, this sentence has too many `or`s in it in my opinion. I would split it into multiple sentences